code/datums/emote.dm

```diff
@@ -0,0 +1,408 @@
+// Emote datums.
+// Check under mob directories for where these get implemented.
+
+// Emote types.
+// These determine how the emote is treated when not directly visible (or audible).
+
+/// Emote is visible. These emotes will be runechatted.
+#define EMOTE_VISIBLE (1<<0)
+/// Emote is audible (in character).
+#define EMOTE_AUDIBLE (1<<1)
+/// Emote makes a sound. These emotes will specifically not be runechatted.
+#define EMOTE_SOUND (1<<2)
+/// Regardless of its existing flags, this emote will not be sent to runechat.
+#define EMOTE_FORCE_NO_RUNECHAT (1<<3)
+/// This emote uses the mouth, and so should be blocked if the user is muzzled or can't breathe (for humans).
+#define EMOTE_MOUTH (1<<4)
+
+// User audio cooldown system.
+// This is a value stored on the user and represents their ability to perform emotes.
+/// The user is not on emote cooldown, and is ready to emote whenever.
+#define EMOTE_READY (1<<0)
+/// The user can spam emotes to their heart's content.
+#define EMOTE_INFINITE (1<<1)
+/// The user cannot emote as they have been blocked by an admin.
+#define EMOTE_ADMIN_BLOCKED (1<<2)
+/// The user cannot emote until their cooldown expires.
+#define EMOTE_ON_COOLDOWN (1<<3)
```

Comment:
any reason these are defined as bitflags when they are never used as bitflags?

Reply:
nope! I'll change that